include/nbl/builtin/hlsl/device_capabilities_traits.hlsl

```diff
@@ -8,11 +8,7 @@
 
 #ifdef __HLSL_VERSION
 
-NBL_GENERATE_MEMBER_TESTER(shaderFloat64);
-NBL_GENERATE_MEMBER_TESTER(subgroupArithmetic);
-NBL_GENERATE_MEMBER_TESTER(fragmentShaderPixelInterlock);
-NBL_GENERATE_MEMBER_TESTER(maxOptimallyResidentWorkgroupInvocations);
-NBL_GENERATE_MEMBER_TESTER(maxResidentInvocations);
+#include "nbl/video/device_capabilities_traits_testers.hlsl"
 
 #define NBL_GENERATE_GET_OR_DEFAULT(field, ty, default) \
 template<typename S, bool = has_member_##field<S>::value> struct get_or_default_##field : integral_constant<ty,S::field> {}; \
```

Comment:
change the `bool = has_member_##field<S>::value` to `has_member_##field<S>::value==(e_member_presence::is_present|e_member_presence::is_static)`


Also write a TODO comment that the `has_member_` needs to be improved because the detectors should really report present, static AND constant

Reply:
comment missing